app/models/package_manager/go.rb

```diff
@@ -243,34 +247,48 @@ def self.project_find_names(name)
 
       begin
         # https://go.dev/ref/mod#serving-from-proxy
-        go_import = get_html("https://#{name}?go-get=1", { request: { timeout: 2 } })
-          .xpath('//meta[@name="go-import"]')
-          .first
-          &.attribute("content")
-          &.value
-          &.split(" ")
-          &.last
-          &.sub(/https?:\/\//, "")
-
-        chosen_name = go_import&.start_with?(*KNOWN_HOSTS) ? [go_import] : [name]
+        # TODO: if nothing is being logged in GO_MODULES_PROJECT_FIND_NAMES_SUCCESS
+        # after a while, we might want to start using root_path instead of
+        # repo_url, if anything.
+        root_path = nil
+        vcs = nil
+        repo_url = nil
```

Comment:
start tracking all 3 values from the "?go-get=1" meta tag, to help prove that this code is not useful